src/bus/bus.h

```diff
@@ -60,3 +60,12 @@ int bus_init(Bus *bus,
 void bus_deinit(Bus *bus);
 
 Peer *bus_find_peer_by_name(Bus *bus, Name **namep, const char *name);
+
+int bus_log_policy_send(Bus *bus, uint64_t sender_id, uint64_t receiver_id, Message *message, const char *file, int line, const char *func);
+int bus_log_policy_receive(Bus *bus, uint64_t sender_id, uint64_t receiver_id, Message *message, const char *file, int line, const char *func);
+
+#define bus_log_policy_send_here(_bus, _sender_id, _receiver_id, _message) \
+        bus_log_policy_send((_bus), (_sender_id), (_receiver_id), (_message), __FILE__, __LINE__, __func__)
+
+#define bus_log_policy_receive_here(_bus, _receiver_id, _sender_id, _message) \
+        bus_log_policy_receive((_bus), (_receiver_id), (_sender_id), (_message), __FILE__, __LINE__, __func__)
```

Comment:
Ugh. Can't we call log_append_here() in the caller? And then, for consistency, call these `bus_log_commit_policy_send()`? Basically following the style of `foo_log_{append,commit}_bar()`.

Reply:
Done.